guides/common/modules/proc_enabling-the-installer-managed-dns-service.adoc

```diff
@@ -0,0 +1,51 @@
+[id="enabling-the-installer-managed-dns-service_{context}"]
+= Enabling the installer-managed DNS service
+
+You can revert to using {ProjectServer} and {SmartProxyServer} as your DNS providers.
+You can use a backup of the answer file that was created before configuring external DNS, or you can create a backup of the answer file.
+ifndef::orcharhino[]
+For more information about answer files, see {InstallingServerDocURL}configuring-server_{project-context}[Configuring {ProjectServer}].
+endif::[]
```

Comment:
Similar notes about "reverting" here. I'd prefer to drop that notion and only treat it as a fresh installation.

Reply:
updated